roles/openshift_logging/files/elasticsearch-logging.yml

```diff
@@ -28,32 +39,24 @@ logger:
 additivity:
   index.search.slowlog: false
   index.indexing.slowlog: false
+  deprecation: false
 
 appender:
-  console:
```

Comment:
Why can't we add an ansible script that modifies the configmap to add this back?